build.gradle

```diff
@@ -11,7 +11,7 @@ apply plugin: 'net.minecraftforge.gradle.forge'
 //Only edit below this line, the above code adds and enables the necessary things for Forge to be setup.
 
 
-version = "1.3.1"
+version = "1.4.0"
```

Comment:
Please dont change the version number, I'll change this when it's ready for a new update.